test/cri-containerd/main.go

```diff
@@ -22,31 +21,42 @@ const (
 	wcowProcessRuntimeHandler         = "runhcs-wcow-process"
 	wcowHypervisorRuntimeHandler      = "runhcs-wcow-hypervisor"
 	wcowHypervisor17763RuntimeHandler = "runhcs-wcow-hypervisor-17763"
+	wcowHypervisor18362RuntimeHandler = "runhcs-wcow-hypervisor-18362"
 	lcowRuntimeHandler                = "runhcs-lcow"
 	imageLcowK8sPause                 = "k8s.gcr.io/pause:3.1"
 	imageLcowAlpine                   = "docker.io/library/alpine:latest"
 	imageLcowCosmos                   = "cosmosarno/spark-master:2.4.1_2019-04-18_8e864ce"
 )
 
 var (
-	relID                       = getHostReleaseID()
-	imageWindowsNanoserver      = "mcr.microsoft.com/windows/nanoserver:" + relID
-	imageWindowsServercore      = "mcr.microsoft.com/windows/servercore:" + relID
-	imageWindowsNanoserver17763 = "mcr.microsoft.com/windows/nanoserver:1809"
+	imageWindowsNanoserver      = getWindowsNanoserverImage(osversion.Get().Build)
+	imageWindowsServercore      = getWindowsServerCoreImage(osversion.Get().Build)
+	imageWindowsNanoserver17763 = getWindowsNanoserverImage(osversion.RS5)
+	imageWindowsNanoserver18362 = getWindowsNanoserverImage(osversion.V19H1)
+	imageWindowsServercore17763 = getWindowsServerCoreImage(osversion.RS5)
+	imageWindowsServercore18362 = getWindowsServerCoreImage(osversion.V19H1)
 )
 
-func getHostReleaseID() string {
-	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\Windows NT\CurrentVersion`, registry.QUERY_VALUE)
-	if err != nil {
-		fmt.Println(err)
+func getWindowsNanoserverImage(build uint16) string {
+	if build == osversion.RS5 {
```

Comment:
Use

```go
switch build {
case 1:
case 2:
default:
    panic("unspported build")
}
```

This is all test's we control so if it panics its 100% ok since we shouldnt use an invalid build